ui/tests/unit/utils/path-tree-test.js

```diff
@@ -0,0 +1,10 @@
+import pathTree from 'nomad-ui/utils/path-tree';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | path-tree', function () {
+  // TODO: Replace this with your real tests.
```

Comment:
Big TODO here!

Reply:
Yup! This is going to be super helpful for documenting this feature.